caesars-palace/scripts/scripts.js

```diff
@@ -16,17 +15,6 @@ import {
 const LCP_BLOCKS = []; // add your LCP blocks to the list
 window.hlx.RUM_GENERATION = 'caesars-palace'; // add your RUM generation information here
 
-function buildHeroBlock(main) {
```

Comment:
We don't want any hero auto blocking done? Can the `hero` block be removed too or is there a plan to still use that block?

Reply:
I posted a comment in Slack when I removed the auto-blocking feature.  My thought is that if we're not going to have anything auto-blocking, we don't need the code.  We can easily pull it back in, should we find a compelling reason.

Not sure about removing the `hero` itself.

Reply:
Sounds good. We can always add it back it we find a need to auto block something.

Reply:
I'm leaning towards deleting `hero` too if it just going to be a dead block we no longer use. That can wait until a litlte later though.

Reply:
+1, I'd wait too